cache/filter.go

```diff
@@ -37,6 +37,21 @@ func AuthorFilter(query string) Filter {
 		return author.Match(query)
 	}
 }
+// MetadataFilter return a Filter that match a bug metadata
+func MetadataFilter(query string) Filter {
+	return func(excerpt *BugExcerpt, resolver resolver) bool {
+		tokens := strings.Split(query, "=")
```

Comment:
At this point of the code, you should already have your string processed into a key and value. Splitting around `=` is a matter for the CLI layer.

Reply:
Aha, and that means I can even do better error handling. :-)